main.md

```diff
@@ -543,6 +543,8 @@ grant_type=authorization_code&code=SplxlOBeZQQYbYS6WxSbIA
 Using the example given above, this request would result in the assignment of the `payment_initiation` authorization details object from (#authz_details) to the access token to be issued (see below).
 
 # Token Response
+The authorization details assigned to the access token issued in a token response are determined by the `authorization_detail` and `resource` parameters of the corresponding token request.  If the client does not specify any of those token request parameters, the AS determines those authorization details at its discretion.
```

Comment:
Suggest that instead of calling out the `resource` parameter specifically like this, we instead generally say that other parameters can influence the processed result of the parameter, like:

```suggestion
The authorization details assigned to the access token issued in a token response are determined by the `authorization_detail` parameter of the corresponding token request as well as any related parameters such as `resource` and `scope`.  If the client does not specify any of those token request parameters, the AS determines the resulting authorization details at its discretion.
```

Reply:
I wouldn't count on reader continuity between adjacent sections, especially at different levels of the heirarchy. Since this paragraph is starting a new top-level major section I would expect it to set expectations on its own.

To me this text is basically saying that the result of the request is the combination of all the different parameters. We're telling people explicitly how to process `authorization_details` but they need to be aware that the others also exist.